tm2/pkg/sdk/auth/ante.go

```diff
@@ -393,6 +393,10 @@ func EnsureSufficientMempoolFees(ctx sdk.Context, fee std.Fee) sdk.Result {
 			if fgd == gpd {
 				prod1 := big.NewInt(0).Mul(fga, gpg) // fee amount * price gas
 				prod2 := big.NewInt(0).Mul(fgw, gpa) // fee gas * price amount
+				// This is equivalent to checking
+				// That the Fee / GasWanted ratio is greater than or equal to the minimum GasPrice per gas.
+				// This approach helps us avoid dealing with configurations where the value of
+				// the minimum gas price is set to 0.00001ugnot/gas.
```

Comment:
This is not outdated. It explains the purpose of the code below the comments, helping us understand the reasoning behind its implementation.

Reply:
first, let's rename gp* to rmg*, and fg* to txg*, since "gas price" is ambiguous, so is "fee"; they are both about gas prices, and both about fees. But one is about the WANTED fee gas price of the TX, while the other is about the fee gas price required MINIMUM. 

better wording: 
// We want to check for txga / txgw >= rmga / rmgg, but this equation can be transformed precisely to avoid rounding errors by checking for txga * rmgg >= rmga * txgw.